fulfillment.go

```diff
@@ -0,0 +1,150 @@
+	TrackingNumbers []string   `json:"tracking_numbers,omitempty"`
+	TrackingUrl     string     `json:"tracking_url,omitempty"`
+	TrackingUrls    []string   `json:"tracking_urls,omitempty"`
+	Receipt         Receipt    `json:"receipt,omitempty"`
+	LineItems       []LineItem `json:"line_items,omitempty"`
+	NotifyCustomer  bool       `json:"notify_customer,omitempty"`
+}
+
+// Receipt represents a Shopify receipt.
+type Receipt struct {
+	TestCase      bool   `json:"testcase,omitempty"`
+	Authorization string `json:"authorization,omitempty"`
+}
+
+// FulfillmentResource represents the result from the fulfillments/X.json endpoint
+type FulfillmentResource struct {
+	Fulfillment *Fulfillment `json:"fulfillment"`
+}
+
+// FulfillmentsResource represents the result from the fullfilments.json endpoint
+type FulfillmentsResource struct {
+	Fulfillments []Fulfillment `json:"fulfillments"`
+}
+
+// List fulfillments
+func (s *FulfillmentServiceOp) List(options interface{}) ([]Fulfillment, error) {
+	prefix := FulfillmentPathPrefix(s.resource, s.resourceID)
+	path := fmt.Sprintf("%s.json", prefix)
+	resource := new(FulfillmentsResource)
+	err := s.client.Get(path, resource, options)
+	return resource.Fulfillments, err
+}
+
+// Count fulfillments
+func (s *FulfillmentServiceOp) Count(options interface{}) (int, error) {
+	prefix := FulfillmentPathPrefix(s.resource, s.resourceID)
+	path := fmt.Sprintf("%s/count.json", prefix)
+	return s.client.Count(path, options)
+}
+
+// Get individual fulfillment
+func (s *FulfillmentServiceOp) Get(fulfillmentID int, options interface{}) (*Fulfillment, error) {
+	prefix := FulfillmentPathPrefix(s.resource, s.resourceID)
+	path := fmt.Sprintf("%s/%d.json", prefix, fulfillmentID)
+	resource := new(FulfillmentResource)
+	err := s.client.Get(path, resource, options)
+	return resource.Fulfillment, err
+}
+
+// Create a new fulfillment
+func (s *FulfillmentServiceOp) Create(fulfillment Fulfillment) (*Fulfillment, error) {
+	prefix := FulfillmentPathPrefix(s.resource, s.resourceID)
+	path := fmt.Sprintf("%s.json", prefix)
+	wrappedData := FulfillmentResource{Fulfillment: &fulfillment}
+	resource := new(FulfillmentResource)
+	err := s.client.Post(path, wrappedData, resource)
+	return resource.Fulfillment, err
+}
+
+// Update an existing fulfillment
+func (s *FulfillmentServiceOp) Update(fulfillment Fulfillment) (*Fulfillment, error) {
+	prefix := FulfillmentPathPrefix(s.resource, s.resourceID)
+	path := fmt.Sprintf("%s/%d.json", prefix, fulfillment.ID)
+	wrappedData := FulfillmentResource{Fulfillment: &fulfillment}
+	resource := new(FulfillmentResource)
+	err := s.client.Put(path, wrappedData, resource)
+	return resource.Fulfillment, err
+}
+
+// Complete an existing fulfillment
+func (s *FulfillmentServiceOp) Complete(fulfillmentID int) (*Fulfillment, error) {
+	prefix := FulfillmentPathPrefix(s.resource, s.resourceID)
+	path := fmt.Sprintf("%s/%d.json", prefix, fulfillmentID)
```

Comment:
I think this needs to be:
`path := fmt.Sprintf("%s/%d/complete.json", prefix, fulfillmentID)`